### ISSUE TITLE
Incorrect --version reported for 2.15.0

### ISSUE THREAD

**Issue by markuspeloquin** [dart-lang/sdk#47884]
Looks like this commit https://github.com/dart-lang/sdk/commit/55134ad5c85d573a39a7d3009006fdb85130c35d that changed the version to 2.16.0 got tagged as 2.15.0.

After building 2.15.0 from source (`git checkout 2.16.0 && gclient sync -D && tools/build.py --mode release --arch x64 --no-goma create_sdk`):

```
% dart --version
Dart SDK version: 2.16.0-edge.da9b3409e6a546202700457d33b56b922c099abc (be) (Wed Dec 8 12:32:30 2021 +0000) on "linux_x64"
```

**Comment by athomas** [comment]
Thanks for the report, this was caused by a bug in the release script that tagged the wrong commit. However, the binaries available on dart.dev and other channels as well as via Flutter should all have the correct version. We will fix the tag to point to 3d2629cd0310df74bdf260d828ed3c75f25db889 instead.

**Comment by athomas** [comment]
Fixed the tag on dart.googlesource.com and removed the incorrect tag on GH. It should re-appear soon.
